packpack

```diff
@@ -127,6 +127,7 @@ chmod a+x ${BUILDDIR}/userwrapper.sh
 #
 env | grep -E "^PRODUCT=|^VERSION=|^RELEASE=|^ABBREV=|^TARBALL_|^CHANGELOG_|^CCACHE_|^PACKAGECLOUD_|^SMPFLAGS=|^OS=|^DIST=" \
     > ${BUILDDIR}/env
+env | grep -E "^PACKPASS_" | sed 's/^PACKPASS_//g' >> ${BUILDDIR}/env
```

Comment:
Nit: you use subset of basic and extended syntax here, so -E option is redundant.